sdk/js/src/api/stream/stream-node.js

```diff
@@ -68,6 +69,11 @@ export default async function(payload, url) {
     },
   })
     .then(response => response.data)
+    .catch(function(error) {
```

Comment:
Can you confirm that the issue occurs when using events in node env?

Reply:
The issue was occurring there as well, the reason being the missing `Content-Type` header.